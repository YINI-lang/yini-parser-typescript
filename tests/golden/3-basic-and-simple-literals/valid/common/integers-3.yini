^ Numbers
int9 = 9223372036854775807     // Max 64-bit signed int
int10 = -9223372036854775808   // Min 64-bit signed int
int11 = 1000000000000000000000 // 10^21 (large, for overflow checks)
int12 = -1000000000000000000000 // -10^21
