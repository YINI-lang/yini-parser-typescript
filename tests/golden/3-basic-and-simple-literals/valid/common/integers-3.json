{
    "Numbers": {
        "int9": 9223372036854776000,
        "int10": -9223372036854776000,
        "int11": 1e+21,
        "int12": -1e+21
    }
}
